eyepop/compute/status.py

```
@@ -46,58 +42,53 @@ async def wait_for_session(
+
+                last_message = f"Pipeline status: {status.value}"
+                await asyncio.sleep(interval)
 
         except ComputeHealthCheckException:
             raise
         except aiohttp.ClientResponseError as e:
             last_message = f"HTTP {e.status}: {e.message}"
-            log.debug(f"HTTP error during health check: {last_message}")
+            log.debug(f"GET /health - error: {last_message} (attempt {attempt})")
+            await asyncio.sleep(interval)
         except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)